src/components/app/Details.js

```diff
@@ -19,16 +19,19 @@ import FlameGraph from '../flame-graph/';
 import { JsTracer } from '../js-tracer/';
 import { selectSidebar } from '../sidebar';
 
-import { changeSelectedTab, changeSidebarOpenState } from '../../actions/app';
-import { getSelectedTab } from '../../selectors/url-state';
-import { getIsSidebarOpen } from '../../selectors/app';
-import { selectedThreadSelectors } from '../../selectors/per-thread';
+import {
+  changeSelectedTab,
+  changeSidebarOpenState,
+} from 'firefox-profiler/actions/app';
+import { getSelectedTab } from 'firefox-profiler/selectors/url-state';
+import { getIsSidebarOpen } from 'firefox-profiler/selectors/app';
+import { selectedThreadSelectors } from 'firefox-profiler/selectors/per-thread';
 import CallNodeContextMenu from '../shared/CallNodeContextMenu';
 import { MaybeMarkerContextMenu } from '../shared/MarkerContextMenu';
```

Comment:
And here:
```
import CallNodeContextMenu from '../shared/CallNodeContextMenu';
import { MaybeMarkerContextMenu } from '../shared/MarkerContextMenu';
```

Reply:
Hi @canova There were module not found errors when attempting to add the firefox-profiler alias to those paths in these 2 files. It seems that the alias is only working for paths that are 2 directories up ```'../..'```, but not 1 ```'../'```, so I need to investigate why this is happening.

Reply:
It can be because you need to append their full path, like `firefox-profiler/components/shared/...`. Note the `components` after the firefox-profiler.